src/components/Product/Product.tsx

```diff
@@ -1,9 +1,32 @@
 import { ProductSchema } from "@/schemas/ProductSchema";
+import { FaRegHeart } from "react-icons/fa6";
 
 interface ProductProps {
   product: ProductSchema;
 }
 
 export const Product = ({ product }: ProductProps) => {
-  return <li>{product.title}</li>;
+  return (
+    <div className="mx-auto mt-11 w-80 transform overflow-hidden rounded-lg bg-white shadow-md duration-300 hover:scale-105 hover:shadow-lg">
```

Comment:
I would center this using `justify-center` or `align-center` in the parent instead of `mx-auto`.